R/has_newline_at_end.R

```diff
@@ -9,7 +9,7 @@ file_has_newline_at_end <- function(filepath) {
   if (!is_file_accessible(filepath) || !is_text_file(filepath)) {
     return(NA)
   }
-  if (.Platform$OS.type == "unix") {
+  if (FALSE) {
```

Comment:
using a logical as a conditional doesn't make much sense

Reply:
Thanks for flagging. This was my way to force the old code to run on my mac.

Reply:
I still don't understand this. It will never run, regardless of the operating system, right? Was this used just for testing?

At a minimum I think this needs some good explanation in an inline comment if it can't be removed.

Reply:
Resolved. That was mistakenly committed during manual testing.